### PR TITLE
Add a github action to run womtool validation on all WDLs

### PR DESCRIPTION
**Pull request by gbggrant**
Added a github action to run womtool validation on all WDLs
Also fixed two wdls that were failing validation.

### CODE REVIEW

File: scripts/cnv_wdl/germline/joint_call_exome_cnvs.wdl [hunk 1]
```diff
@@ -151,7 +151,7 @@ workflow JointCallExomeCNVs {
     }
     Array[String] subset_arr = select_all(subset)
     Array[String] subset_index_arr = select_all(subset_indexes)
-    Array[String] failed_samples = select_all(failed)
+    Array[String] failed_samples_arr = select_all(failed)
```
Comment (gbggrant) [inline scripts/cnv_wdl/germline/joint_call_exome_cnvs.wdl:154]: Changed the name of this (internal) variable as it is also defined as an output and was failing validation for that reason

File: scripts/variantstore/tieout/legacy_wdl/JointGenotypingTasks.wdl [hunk 2]
```diff
@@ -0,0 +1,1070 @@
+version 1.0
```
Comment (gbggrant) [inline scripts/variantstore/tieout/legacy_wdl/JointGenotypingTasks.wdl:1]: Completely added this file to make JointGenotyping.wdl validate (it couldn't find the file, which is zipped up in a dependencies zip).

File: .github/workflows/womtool_check.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+jobs:
+  validate_wdl_job:
+    runs-on: ubuntu-latest
+    name: Validate all wdl using womtool
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: broadinstitute/gatk
+      - name: Run womtools
```
Comment (mcovarr) [inline .github/workflows/womtool_check.yml:19]: nit
```suggestion
      - name: Run womtool validate
```

**Review by mcovarr — APPROVED** [review]
Awesome, thank you for adding this!

**Pull request closed without merging**